presto-main/src/main/java/com/facebook/presto/execution/buffer/SerializedPage.java

```diff
@@ -74,6 +104,7 @@ public Slice getSlice()
         return slice;
     }
 
+    @ThriftField(2)
```

Comment:
nit: Should we consider reordering these methods given the field id seems to be out of order 😂

Reply:
+1